assets/js/components/BatchEdit/About/About.jsx

```diff
@@ -92,43 +78,76 @@ const BatchEditAbout = ({ numberOfResults }) => {
         </span>
         You are editing {numberOfResults} items. Proceed with caution.
       </p>
+      <UIAccordion
```

Comment:
This `UIAccordion`s are great.  Curious when would the `isVisible` prop be `false`?

Reply:
`isVisible` is always true for the way we're using it but I thought if we have to use it so that the accordions load as closed by default, we could use the prop.

Reply:
Ah, now I see what you mean.  To simplify this, what do you think about changing the `isVisible` prop (to me that sounds like the whole element, not necessarily the contents of the accordion component) to `defaultOpen`, which is a prop that defaults to `true`.   If a user wanted to render the component initially closed, then they could include the prop `defaultOpen={false}`?